portal-bridge/src/types/mode.rs

```diff
@@ -8,6 +8,8 @@ use trin_validation::constants::EPOCH_SIZE;
 ///   - ex: "e123" starts at epoch 123
 /// - Single: executes a single block
 ///   - ex: "b123" executes block 123
+/// - BlockRange: generates test data from block x to y
+///   - ex: "r10-12" starts from block 10 and goes till 12
```

Comment:
Same nitpick about `till`, and I do think it's useful to be explicit about whether or not the range is inclusive in these docstrings